react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.resetTimer = this.resetTimer.bind(this);
+    this.state = {
+      timer: 0,
+      isRunning: false
+    }
+  }
+  handleClick() {
+    if(!this.state.isRunning) {
+      this.setState({isRunning: true});
+      this.timerID = setInterval(() => {
+        this.setState({timer: this.state.timer + 1})
+      }, 1000)
+    } else {
+      this.setState({isRunning: false});
+      clearInterval(this.timerID);
+    }
+  }
+  resetTimer() {
+    if(!this.state.isRunning) {
+      this.setState({timer: 0})
+    }
+  }
+  render() {
+    return (
+      <div className="container">
+        <div onClick={this.resetTimer} className="circle">
+          <div className="number">{this.state.timer}</div>
+        </div>
+        <div onClick={this.handleClick} className={this.state.isRunning ? this.props.start.pause : this.props.start.play}></div>
+      </div>
+    );
+  }
+}
+
+export const pausePlay = {
```

Comment:
Very clever use of an exported object so that you can use the values as props being passed to the component. with that said, it would be best to store this information in the constructor of your class at properties so that you can access them directly from the render method as needed. 

I also want to say that using props to configure components is a huge part of React and you are demonstrating that skill, its just that in this instance, this isolated component does not need to receive outside. info, as the display of the icons should be part of the components core functionality which does not require outside data.